docs/modules/sql/pages/alter-job.adoc

```diff
@@ -70,5 +75,14 @@ You may want to restart a job if you want to distribute it over some new members
 ----
 ALTER JOB track_trades RESTART
 ----
+[[update]]
+=== Update the Configuration of a Suspended Job
 
+For troubleshooting, you may want to update the configuration of a failed job to see if processing fewer items allows the job to run successfully.
```

Comment:
Fewer or more? And again this isn't done for troubleshooting. The problem is generally understood by error messages and the solution might be to increase the accumulation limit.

Reply:
See comment above about troubleshooting.

Reply:
Updated as requested.